Formula/y/yazi.rb

```diff
@@ -0,0 +1,28 @@
+class Yazi < Formula
+  desc "️Blazing fast terminal file manager written in Rust, based on async I/O"
+  homepage "https://github.com/sxyazi/yazi"
+  url "https://github.com/sxyazi/yazi/archive/refs/tags/v0.1.3.tar.gz"
+  sha256 "e6738a12896ff0ab081a306e6334702a93dcb1039e4c596eb6ee723a6bc037cc"
+  license "MIT"
+  head "https://github.com/sxyazi/yazi.git",
+       branch: "main"
+
+  depends_on "rust" => :build
+  depends_on "fd"
+  depends_on "ffmpegthumbnailer"
+  depends_on "fzf"
+  depends_on "jq"
+  depends_on "poppler"
+  depends_on "ripgrep"
+  depends_on "unar"
+  depends_on "zoxide"
```

Comment:
That would pull in [a significantly large number of dependencies](https://github.com/Homebrew/homebrew-core/actions/runs/5893045728/job/15983871606?pr=139792#step:4:29). I don't suggest adding them here as [they are optional](https://github.com/sxyazi/yazi#installation).

Reply:
Remove them